medcat/cat.py

```diff
@@ -717,8 +719,8 @@ def train_supervised(self,
                          never_terminate: bool = False,
                          train_from_false_positives: bool = False,
                          extra_cui_filter: Optional[Set] = None,
-                         checkpoint: Optional[Checkpoint] = None,
-                         resume_from_checkpoint: bool = False) -> Tuple:
```

Comment:
In which case would someone provide a checkpoint but set is_resumed to False?

Reply:
The user may want to configure the checkpoint with customised values before starting a new training. Having both parameters also make unit tests easier.

Reply:
I do not get it, can you please give me an example for the first reason. In fact, also for the second.

Reply:
Does not make any sense to me. But, since Tom has already approved, we can close this.

Reply:
We can always get rid of this optional argument if proved useless. For now it is more for testability and advanced usage.